package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@glif/modules",
   "private": false,
   "scripts": {
-    "bootstrap": "lerna bootstrap",
+    "bootstrap": "lerna bootstrap --no-ci",
```

Comment:
We might test turning off --no-ci and test out --hoist.

Reply:
Yeah ,i think as of now you have to use `--hoist` (although not sure what --no-ci does)? Looking at some of the other packages, the commands like test and build point directly to the node_modules at the repo root. If those don't end up getting hoisted, I'm pretty sure those scripts will break. For example https://github.com/glifio/modules/pull/22/files#diff-2b01dbf04a098f3ee9dcfa5163056e971f37eedb9ef53b0d838965516347a222R17

Reply:
I will tweak once everything is working.